RUFAS/biophysical/animal/animal_config.py

```
@@ -569,3 +602,24 @@ def initialize_animal_config(cls) -> None:
         cls.feedlot_ndf_minimum_pct = float(feedlot_cfg.get("ndf_minimum_pct", 10.0))
+
+        # ── COW-CALF PARAMETERS ──────────────────────────────────────────────
+        beef_cfg: dict[str, Any] = animal_config_data.get("beef_cow_calf", {})
+        cls.beef_breeding_season_start_day = int(beef_cfg.get("breeding_season_start_day", 90))
+        cls.beef_breeding_season_length = int(
+            beef_cfg.get("breeding_season_length", AnimalModuleConstants.BEEF_DEFAULT_BREEDING_SEASON_LENGTH_DAYS)
+        )
+        cls.beef_weaning_age_days = int(
+            beef_cfg.get("weaning_age_days", AnimalModuleConstants.BEEF_DEFAULT_WEANING_AGE_DAYS)
+        )
+        raw_weaning_weight = beef_cfg.get("weaning_weight_kg", None)
```

Ruff (0.15.17)

[warning] Use `beef_cfg.get("weaning_weight_kg")` instead of `beef_cfg.get("weaning_weight_kg", None)`

Replace `beef_cfg.get("weaning_weight_kg", None)` with `beef_cfg.get("weaning_weight_kg")`

(SIM910)